mmcv/runner/hooks/lr_updater.py

```diff
@@ -212,5 +213,70 @@ def get_lr(self, runner, base_lr):
         else:
             progress = runner.iter
             max_progress = runner.max_iters
-        return self.target_lr + 0.5 * (base_lr - self.target_lr) * \
-            (1 + cos(pi * (progress / max_progress)))
+        if self.as_ratio:
+            target_lr = base_lr * self.target_lr
+            return target_lr + 0.5 * (base_lr - target_lr) * \
+                (1 + cos(pi * (progress / max_progress)))
+        else:
+            return self.target_lr + 0.5 * (base_lr - self.target_lr) * \
+                (1 + cos(pi * (progress / max_progress)))
+
+
+class CyclicLrUpdaterHook(LrUpdaterHook):
+
+    def __init__(self,
+                 by_epoch=False,
+                 target_ratio=[10, 1e-4],
+                 cyclic_times=1,
+                 step_ratio_up=0.4,
+                 **kwargs):
+        if isinstance(target_ratio, float):
+            target_ratio = [target_ratio, target_ratio / 1e5]
+        elif isinstance(target_ratio, list):
+            target_ratio = (
+                target_ratio + target_ratio[0] / 1e5
+                if len(target_ratio) == 1 else target_ratio)
```

Comment:
If `len(target_ratio) == 1`, then `target_ratio = target_ratio + target_ratio[0] / 1e5`.
However, `target_ratio` is a list, `target_ratio[0]` is a float number, I guess this will report an error.